psi4/src/psi4/lib3index/dfhelper.h

```diff
@@ -105,6 +105,12 @@ class PSI_API DFHelper {
     void set_AO_core(bool core) { AO_core_ = core; }
     bool get_AO_core() { return AO_core_; }
 
+    /// Switch flag for releasing in-core AO integrals to store 
+    /// intermediates for the metric contraction. Used in methods 
+    /// that AO integrals are not needed in the contraction step 
+    /// like SAPT(DFT). 
+    void switch_release_AO() { release_AO_ = true; }
```

Comment:
I don't know if `release_AO` is best set up. Default is false, and this switch always changes it to true. But the fn name is just "switch" which could imply `release_AO_ = ! release_AO_`. Perhaps a regular setter would be clearer.

Reply:
Thanks, changed it to a setter. It might be an issue in English since "switch" is what pops up in my head when trying to do this :)